README.md

```diff
@@ -16,18 +16,18 @@ Add the **Deno** binary to the PATH variable in your shell configuration file.
 For ZSH shell, for instance:
 
 ```sh
-echo 'export DENO_INSTALL="/Users/yourUser/.deno"' >> ~/.zshrc
+echo 'export DENO_INSTALL="/Users/$USER/.deno"' >> ~/.zshrc
 echo 'export PATH="$DENO_INSTALL/bin:$PATH"' >> ~/.zshrc
 ```
 
 Install **Velociraptor** to watch files, list available scripts, and run
 predefined scripts:
 
 ```sh
-deno install -qA -n vr https://deno.land/x/velociraptor/cli.ts
+deno install -qA -n vr https://deno.land/x/velociraptor@1.1.0/cli.ts
```

Comment:
is this working?

---
**Sema Reaction:** :question: I have a question

Reply:
I checked, so it is

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
Yep! I tested it before commiting.